### PR TITLE
feat: Match design guidelines for topbar and transaction settings popover

### PR DESCRIPTION
**Pull request by JoseRFelix**

### CODE REVIEW

**Review by jonator — COMMENTED** [review]
Seems this is using the old pools query that is no longer supported. I'd update from stage to fix.

File: packages/web/components/popover.tsx [hunk 1]
```diff
@@ -0,0 +1 @@
+export { Popover } from "@headlessui/react";
```
Comment (jonator) [inline packages/web/components/popover.tsx:1]: What's the purpose of this file?

**Review by jonator — CHANGES_REQUESTED** [review]
Can we make sure the background of the settings popover and the drawer are the same color?

Screen Shot 2023-01-22 at 7 16 48 PM
Screen Shot 2023-01-22 at 7 17 20 PM

File: packages/web/components/popover.tsx (hunk 1 above)
Comment (JoseRFelix) [reply packages/web/components/popover.tsx:1]: Most likely, we'll extend the popover functionality in the future by adding animations. So I think it's worth using our own import to not have to replace all imports.

**Review by jonator — APPROVED** [review]

**Review by jonator — APPROVED** [review]

**Comment by jonator** [comment]
@JoseRFelix please fix fix merge conflicts once you get the chance, then I can include this in the next staged bundle

**Pull request merged**